.github/scripts/windows/test_task.bat

```diff
@@ -91,16 +92,16 @@ rem work-around for failing to dl(mysqli) with OPcache (https://github.com/php/p
 if "%OPCACHE%" equ "1" set OPCACHE_OPTS=%OPCACHE_OPTS% -d extension=mysqli
 
 rem prepare for enchant
-mkdir %~d0\usr\local\lib\enchant-2
+if NOT EXIST %SCRIPT_DRIVE%\usr\local\lib\enchant-2 mkdir %SCRIPT_DRIVE%\usr\local\lib\enchant-2
 if %errorlevel% neq 0 exit /b 3
-copy %DEPS_DIR%\bin\libenchant2_hunspell.dll %~d0\usr\local\lib\enchant-2
+copy %DEPS_DIR%\bin\libenchant2_hunspell.dll %SCRIPT_DRIVE%\usr\local\lib\enchant-2
 if %errorlevel% neq 0 exit /b 3
-mkdir %~d0\usr\local\share\enchant\hunspell
+if NOT EXIST %SCRIPT_DRIVE%\usr\local\share\enchant\hunspell mkdir %SCRIPT_DRIVE%\usr\local\share\enchant\hunspell
 if %errorlevel% neq 0 exit /b 3
 echo Fetching enchant dicts
-pushd %~d0\usr\local\share\enchant\hunspell
+pushd %SCRIPT_DRIVE%\usr\local\share\enchant\hunspell
 powershell -Command wget http://windows.php.net/downloads/qa/appveyor/ext/enchant/dict.zip -OutFile dict.zip
-unzip dict.zip
+unzip -o dict.zip
 del /q dict.zip
 popd
```

Comment:
If we want to use this locally we should separate installation (i.e. downloading of libs, unzipping, etc.) into a separate script that doesn't have to be run every time PHP is built.